Hibernate/hibernateCRUD/DeleteStudentDemo.java

```diff
@@ -0,0 +1,39 @@
+package hibernateCRUD;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+import hibernateAnnotationDemo.Student;
+
+public class DeleteStudentDemo {
+
+	public static void main(String[] args) {
+		SessionFactory factory = new Configuration()
```

Comment:
Abstract this logic of connecting to database in a separate package may be in a static method of a class